src/views/layouts/editGame.hbs

```diff
@@ -1,74 +1,100 @@
+            {{#streq game.status "Ended"}}
+                <input type="number" id="goals1" name="goals1" value="{{game.goals.[0]}}" required class="form-control" min="0">
+            {{else}}
+                <p class="form-control-plaintext"><b>{{game.goals.[0]}}</b></p>
+            {{/streq}}
+        </div>
 
-        <label for="team2">Team 2:</label><br>
-        {{#streq game.status "Ended"}}
-            {{#each teams}}
-                {{#if (eq this._id (lookup ../game.opponents 1))}}
-                    <p><b>{{this.name}}</b></p>
-                {{/if}}
-            {{/each}}
-        {{else}}
-        <select id="team2" name="team2" required class="form-select mb-2">
-            {{#each teams}}
-                {{#if (eq this._id (lookup ../game.opponents 1))}}
-                    <option value="{{this._id}}" selected>{{this.name}}</option>
-                {{else}}
-                    <option value="{{this._id}}">{{this.name}}</option>
-                {{/if}}
-            {{/each}}
-        {{/streq}}
-        </select><br>
+        <div class="mb-3">
+            <label for="team2" class="form-label text-decoration-underline">Team 2:</label>
+            {{#streq game.status "Ended"}}
+                {{#each teams}}
+                    {{#if (eq this._id ../game.opponents.[1]._id)}}
+                        <p class="form-control-plaintext"><b>{{this.name}}</b></p>
+                    {{/if}}
+                {{/each}}
+            {{else}}
+                <select id="team2" name="team2" required class="form-select">
+                    {{#each teams}}
+                        {{#if (eq this._id ../game.opponents.[1]._id)}}
+                            <option value="{{this._id}}" selected>{{this.name}}</option>
+                        {{else}}
+                            <option value="{{this._id}}">{{this.name}}</option>
+                        {{/if}}
+                    {{/each}}
+                    {{#if game.opponents.[1].isDummy}}
+                        <option value="{{game.opponents.[1].name}}" selected>{{game.opponents.[1].name}}</option>
+                    {{/if}}
+                </select>
+            {{/streq}}
+        </div>
 
-        <label for="goals2">Goals (Team 2):</label><br>
-        {{#streq game.status "Ended"}}
-        <input type="number" id="goals2" name="goals2" value="{{game.goals.[1]}}" required class="form-control mb-2" min="0">
-        {{else}}
-        <p><b>{{game.goals.[1]}}</b></p>
-        {{/streq}}
+        <div class="mb-3">
+            <label for="goals2" class="form-label text-decoration-underline">Tore (Team 2):</label>
+            {{#streq game.status "Ended"}}
+                <input type="number" id="goals2" name="goals2" value="{{game.goals.[1]}}" required class="form-control" min="0">
+            {{else}}
+                <p class="form-control-plaintext"><b>{{game.goals.[1]}}</b></p>
```

Comment:
If the number of goals is changed here sekt and scoring players are not considered - also i think the counters are not updated

Reply:
Thats right nothing but the game is updated - but ist dificult to to update the goals because you don't know exactly which goals was changed/removed when editing under normal circumstances i hope its not neccesary to edit a game after its finished - thats a problem every time